specify/specify7
# Remove the prompt field option from weblinks

Fixes #4525

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions
 go to weblink editor 
verify that the feature "Prompt Field" is not available anymore
verify that the other features are still working
